### PR TITLE
Fix master failure for password reset tests

### PR DESCRIPTION
**Pull request by munnfaye-deriv**
**Card**
https://app.clickup.com/t/20696747/QAA-1667

**Description**
Fix master failure for password reset tests
image
image (1)

### CODE REVIEW

File: cypress/e2e/smoke/UPM/passwordReset/resetPasswordLoggedOut.cy.js [hunk 1]
```diff
@@ -18,30 +18,31 @@ describe('QATEST-173757: Verify user should be able to reset password when logge
...
+        `${Cypress.env('derivComProdURL')}reset-password`,
         size
       )
-      cy.findByRole(
-        'button',
-        { name: 'whatsapp icon' },
-        { timeout: 30000 }
-      ).should('be.visible')
-      cy.findByPlaceholderText('example@email.com').clear().type(userEmail)
+      cy.get('input#email-reset-password').clear().type(userEmail)
```
Comment (abuzar-deriv) [inline cypress/e2e/smoke/UPM/passwordReset/resetPasswordLoggedOut.cy.js:45]: suggestion: Can you try
```suggestion
      cy.findByLabelText('Email').clear().type(userEmail)
```

File: cypress/support/commands/common.js [hunk 2]
```diff
@@ -858,24 +860,22 @@ Cypress.Commands.add('c_visitBackOffice', (options = {}) => {
...
     Cypress.env('stdConfigServer'),
-    {
-      args: [email, password, oldPassword],
-    },
-    ([email, password, oldPassword]) => {
-      Cypress.on('uncaught:exception', (err) => {
-        if (err.message.includes(`Unexpected token ','`)) return false
+    { args: { email, password, oldPassword } },
+    function ({ email, password, oldPassword }) {
+      Cypress.on('uncaught:exception', function (err) {
```
Comment (abuzar-deriv) [inline cypress/support/commands/common.js:865]: question: These are just semantic changes. Any particular reason?

File: cypress/support/commands/common.js [hunk 3]
```diff
@@ -858,24 +860,22 @@ Cypress.Commands.add('c_visitBackOffice', (options = {}) => {
...
         'Your email and/or password is incorrect. Perhaps you signed up with a social account?'
       ).should('be.visible')
-      //Successful login
-      cy.get(`input[type='email']`).clear().type(email)
-      cy.get(`input[type='password']`).clear().type(password, { log: false })
-      cy.get(`button[name='login']`).click()
+      // Successful login
+      cy.get('input[type="email"]').clear().type(email)
+      cy.get('input[type="password"]').clear().type(password, { log: false })
+      cy.get('button[name="login"]').click()
```
Comment (abuzar-deriv) [inline cypress/support/commands/common.js:878]: question: Same here; these are just semantic changes. Any particular reason?

**Review by abuzar-deriv — CHANGES_REQUESTED** [review]

File: cypress/support/commands/common.js (hunk 2 above)
Comment (munnfaye-deriv) [reply cypress/support/commands/common.js:865]: Prettier check was failing here

File: cypress/support/commands/common.js (hunk 3 above)
Comment (munnfaye-deriv) [reply cypress/support/commands/common.js:878]: Prettier check was failing here as well

File: cypress/e2e/smoke/UPM/passwordReset/resetPasswordLoggedOut.cy.js (hunk 1 above)
Comment (munnfaye-deriv) [reply cypress/e2e/smoke/UPM/passwordReset/resetPasswordLoggedOut.cy.js:45]: I've tried this but I think it does not work as it is a `class="floating-label"`

**Review by abuzar-deriv — APPROVED** [review]
LGTM

**Review by maliheh-deriv — APPROVED** [review]

**Pull request merged**